fuel-relayer/src/config.rs

```diff
@@ -1,163 +1,54 @@
-
-pub fn keccak256(data: &'static str) -> H256 {
-    let out = Keccak256::digest(data.as_bytes());
-    H256::from_slice(out.as_slice())
-}
-
 pub(crate) static ETH_LOG_MESSAGE: Lazy<H256> =
-    Lazy::new(|| keccak256("SentMessage(bytes32,bytes32,bytes32,uint64,uint64,bytes)"));
-pub(crate) static ETH_LOG_VALIDATOR_REGISTRATION: Lazy<H256> =
-    Lazy::new(|| keccak256("ValidatorRegistration(bytes,bytes)"));
-pub(crate) static ETH_LOG_VALIDATOR_UNREGISTRATION: Lazy<H256> =
-    Lazy::new(|| keccak256("ValidatorUnregistration(bytes)"));
-pub(crate) static ETH_LOG_DEPOSIT: Lazy<H256> =
-    Lazy::new(|| keccak256("Deposit(address,uint256)"));
-pub(crate) static ETH_LOG_WITHDRAWAL: Lazy<H256> =
-    Lazy::new(|| keccak256("Withdrawal(address,uint256)"));
-pub(crate) static ETH_LOG_DELEGATION: Lazy<H256> =
-    Lazy::new(|| keccak256("Delegation(address,bytes[],uint256[])"));
-pub(crate) static ETH_FUEL_BLOCK_COMMITTED: Lazy<H256> =
-    Lazy::new(|| keccak256("BlockCommitted(bytes32,uint32)"));
+    Lazy::new(crate::abi::bridge::message::SentMessageFilter::signature);
 
 #[derive(Clone, Debug)]
+/// Configuration settings for the Relayer.
 pub struct Config {
     /// Number of da block after which messages/stakes/validators become finalized.
     pub da_finalization: DaBlockHeight,
     /// Uri address to ethereum client.
-    pub eth_client: Option<String>,
-    /// Ethereum chain_id.
-    pub eth_chain_id: u64,
-    /// Contract to publish commit fuel block.  
-    pub eth_v2_commit_contract: Option<H160>,
+    pub eth_client: Option<url::Url>,
     /// Ethereum contract address. Create EthAddress into fuel_types.
     pub eth_v2_listening_contracts: Vec<H160>,
-    /// Block number after we can start filtering events related to fuel.
-    /// It does not need to be accurate and can be set in past before contracts are deployed.
-    pub eth_v2_contracts_deployment: DaBlockHeight,
-    /// Number of blocks that will be asked at one time from client, used for initial sync.
-    pub initial_sync_step: usize,
-    /// Refresh rate of waiting for eth client to finish its initial sync.
-    pub initial_sync_refresh: Duration,
+    /// Number of pages or blocks containing logs that
+    /// should be downloaded in a single call to the da layer
+    pub log_page_size: u64,
+    /// This throttles the background relayer loop to
+    /// at least this duration to prevent spamming the DA node.
+    pub sync_minimum_duration: Duration,
+}
+
+#[allow(missing_docs)]
+impl Config {
+    pub const DEFAULT_LOG_PAGE_SIZE: u64 = 5;
+    pub const DEFAULT_DA_FINALIZATION: u64 = 100;
+    pub const DEFAULT_SYNC_MINIMUM_DURATION: Duration = Duration::from_secs(5);
 }
 
 impl Default for Config {
     fn default() -> Self {
         Self {
-            da_finalization: DaBlockHeight::from(64u64),
+            da_finalization: DaBlockHeight::from(Self::DEFAULT_DA_FINALIZATION),
             // Some(String::from("http://localhost:8545"))
```

Comment:
This line should probably be removed or updated since `eth_client` is now a `URL` and not a string